app.py

```diff
@@ -45,9 +46,12 @@ def index():
         #                                     NotamRequest.radius) #page num here is one temporarily
         #                                     for latitude, longitude in coordList ]
         apiOutputs = []
+        #Necessary for converting the time and date to the proper form for working with the API
+        startDate = ZuluConverter.time_converter(NotamRequest.effectiveStartDate, '%Y-%m-%d %H:%M:%S', 'CST')

```

Comment:
is it right to assume user input is always in CST?  Actually, we're currently in CDT now (Daylight Saving Time began this past Sunday morning, can't you tell by all the daylight we've been saving since then? :))

Should this be something the user can specify?  Should this be something the code "detects" based on the system settings?